### ISSUE TITLE
Progress Reporting

### ISSUE THREAD

**Issue by mjpt777** [melix/jmh-gradle-plugin#45]
It would be great if the plugin reported progress of the JMH benchmark that is executing. In the current form it is stuck at 88% and appears like the build is broken to a newbie.

**Comment by vyazelenko** [comment]
Duplicate of #17. Can be closed once pull #33 is merged.

**Comment by vyazelenko** [comment]
Actually #33 was already merged but issue #17 was not closed.
